mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
+    .exec()
+    .then(result => {
+      if (result.n === 0) {
```

Comment:
The condition `if (result.n === 0)` is not descriptive. It would be better to use a more descriptive variable name instead of `n`. For example, `deletedCount` or `numberOfDeletedDocuments` would be more readable.